std/node/_crypto/randomBytes.ts

```diff
@@ -50,9 +52,12 @@ export default function randomBytes(
       ) {
         throw e;
       } else {
-        cb(e);
+        err = e;
       }
     }
+    setTimeout(() => {
+      cb(err, bytes);
+    }, 0);
```

Comment:
```suggestion
    cb(err, bytes);
```
If the setTimeout is there for a reason, can you add an explaining comment? Thanks.

Reply:
Since the callback is called synchronously, any errors it throws can be caught with a try/catch around `randomBytes()`. That isn't really related to this issue, but I originally did it because for some reason [a V8 debug check was failing](https://github.com/denoland/deno/runs/1610754385#step:25:4138) when pbkdf2 was synchronous.